integration/spark-common/src/main/scala/org/apache/carbondata/events/AlterTableEvents.scala

```diff
@@ -131,40 +135,44 @@ case class AlterTableRenameAbortEvent(carbonTable: CarbonTable,
 
 
 /**
- *
+ * Class for handling pre compaction operations
```

Comment:
This is not the class to handle the event, please describe when this event will be triggered. And add description of how it is handled in the corresponding listener